browser/components/StorageList.js

```diff
@@ -0,0 +1,23 @@
+/**
+* @fileoverview Micro component for showing StorageList
+*/
+import React, { PropTypes } from 'react'
+import styles from './StorgaeList.styl'
+import CSSModules from 'browser/lib/CSSModules'
+
+/**
+* @param {Array} storgaeList
+*/
+
+const StorageList = ({storageList}) => (
+  <div styleName='storageList'>
+    {storageList.length > 0 ? storageList : (
+      <div styleName='storgaeList-empty'>No storage mount.</div>
+    )}
+  </div>
+)
+
+StorageList.propTypes = {
+  storgaeList: PropTypes.arrayOf.isRequired
```

Comment:
The way to use `arrayOf` is wrong. Please read the document.
The code below is sample from the document.
`optionalArrayOf: PropTypes.arrayOf(PropTypes.number),`

Reply:
Thank you for telling me.
I fixed it at 83002ad .